packages/use-contractkit/src/modals/connect.tsx

```diff
@@ -53,10 +55,13 @@ export const ConnectModal: React.FC<ConnectModalProps> = ({
   const providers = useMemo<[providerKey: string, provider: Provider][]>(
     () =>
       Object.entries(PROVIDERS).filter(
-        ([, provider]) =>
+        ([providerKey, provider]) =>
           typeof window !== 'undefined' &&
           provider.showInList() &&
-          Object.keys(screens).find((screen) => screen === provider.name)
+          Object.keys(screens).find((screen) => screen === provider.name) &&
+          // eslint-disable-next-line
+          // @ts-ignore cant deduce that the keys are from the SupportedProviders enum
```

Comment:
Yes TS object keys support is a constant source of frustration. I suggest a cast instead of an ignore but nbd